main/input_default.cpp

```diff
@@ -686,7 +686,8 @@ void InputDefault::release_pressed_events() {
 	_joy_axis.clear();
 
 	for (Map<StringName, InputDefault::Action>::Element *E = action_state.front(); E; E = E->next()) {
-		action_release(E->key());
+		if (action_state[E->key()].pressed)
```

Comment:
Since `E` is already an element from `action_state`, wouldn't `E->get().pressed` be the more direct way to access this?

Reply:
Changed.